python/tskit/trees.py

```diff
@@ -5521,7 +5521,8 @@ def write_vcf(
 
```

Comment:
```suggestion

.. note::
   Older code often uses the ``ploidy=2`` argument, because previous versions of msprime did not
   output individual data. Specifying individuals in the tree sequence is more robust, and since
   tree sequences now  typically contain individuals (e.g., as produced by
   ``msprime.sim_ancestry( )``), this is not necessary, and the ``ploidy`` argument can safely be removed
   from old code in most cases.
```